### ISSUE TITLE
annotation guideline

### ISSUE THREAD

**Issue by goPikachu88** [yuhaozhang/tacred-relation#8]
Hi Yuhao,

Is there an annotation guideline for the TACRED dataset? I'm curious about how you define the object entity types and relations.

Thank you in advance.

**Comment by yuhaozhang** [comment]
Hi @ardellelee,

We created TACRED by closely following the guidelines used in the TAC KBP challenge. So I will suggest the following materials:

1) [Our dataset website](https://nlp.stanford.edu/projects/tacred/) includes some basic statistics of the dataset over different relation types;
2) For more detailed definitions of the relations and entity types, please refer to the guideline documents (task definition, slot descriptions, etc.) published by TAC at: http://surdeanu.info/kbp2014/def.php

**Comment by goPikachu88** [comment]
Hi, @yuhaozhang ,

Thank you for providing the resources. It really helps to clarify some of my concerns.

From your website I know that this huge dataset is constructed by crowd sourcing: 

> These examples are created by combining available human annotations from the TAC KBP challenges and crowdsourcing.

May I confirm with you about:
1) Are you using all the testing queries & keys from KBP 2009-2014 as part of the TACRED dataset? How many percent does this part take in the whole dataset? (a rough number can be helpful)
2) Is there any quality control over the crowdsourcing part? Like, what is the agreement among annotators?  is there any gold Precision/Recall/F1?  (such as the LDC score in the KBP overview slides each year)

Since recently there are some solutions [using pre-trained language models on this dataset](https://github.com/DFKI-NLP/TRE), I'm wondering if they are proximate to the upper limit, or already exceed human performance.

Thanks!

**Comment by yuhaozhang** [comment]
Hi @ardellelee, here are my answers:

1. Yes we did use all queries when collecting the dataset. That said, it is not guaranteed that all queries occur in the final dataset. It is entirely possible that examples of some queries were dropped during the collection process, due to quality issues, etc. Unfortunately I cannot provide an exact number of how many percent are left in the final dataset.

2. We did employ various forms of quality control. I have included details on this in the Appendix A.2 of [the original paper](https://nlp.stanford.edu/pubs/zhang2017tacred.pdf).

Hope these are helpful!
